src/OrchardCore/OrchardCore.Contents.Core/ContentSearchOption.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+
+namespace OrchardCore.Contents;
+
+public class ContentSearchOption
+{
+    private readonly Dictionary<string, string> _maps = new(StringComparer.OrdinalIgnoreCase);
```

Comment:
Do we really need to be case insensitive ;)

Reply:
Yes because we'll read the selectedContentType from the UI "URL" or filter.